### PR TITLE
ISS-23 deck - add attributes owner and public

### LINKED ISSUE #23: database - add owner of decks and boolean public= True or False

### PR DESCRIPTION
**Pull request by Hamilkar247**
fixes #23

### CODE REVIEW

**Comment by Hamilkar247** [comment]
I 
 fix owner = models.ForeignKey("auth.User", related_name="deck", on_delete=models.CASCADE, null=True)
to
 owner = models.ForeignKey("auth.User", related_name="owner", on_delete=models.CASCADE, null=True)

File: core/models.py [hunk 1]
```diff
@@ -6,10 +6,15 @@ class Deck(models.Model):
     description = models.CharField(max_length=1200, verbose_name="Description of deck")
     tag = models.CharField(max_length=50, verbose_name="tag")
     flashCard = models.ManyToManyField('FlashCard', related_name='flashCard', through='DeckFlashCard')
+    public = models.BooleanField(default=False)
+    owner = models.ForeignKey("auth.User", related_name="owner", on_delete=models.CASCADE, null=True)
```
Comment (jlatusek) [inline core/models.py:10]: As I understand this response: https://stackoverflow.com/questions/2642613/what-is-related-name-used-for-in-django
related_name in this situation supposed to be deck

File: core/models.py [hunk 2]
```diff
@@ -6,10 +6,15 @@ class Deck(models.Model):
...
 
     def __str__(self):
         return str(self.name)
 
+    def save(self, *args, **kwargs):
+        super(Deck, self).save(*args, **kwargs)
```
Comment (jlatusek) [inline core/models.py:16]: why override default save method and don't change anything?

Comment (Hamilkar247) [reply core/models.py:16]: Effect of copying from django rest api

File: core/models.py (hunk 1 above)
Comment (Hamilkar247) [reply core/models.py:10]: Hmm and what do you thing about other related_name in foreign keys in models.py file?

Comment (Hamilkar247) [reply core/models.py:10]: ok now i understand

Comment (Hamilkar247) [reply core/models.py]: changed to deck

File: core/models.py (hunk 2 above)
Comment (Hamilkar247) [reply core/models.py]: deleted

**Comment by Hamilkar247** [comment]
ok, can you merge this?

**Pull request merged**